moped-editor/src/app/App.jsx

```diff
@@ -7,15 +7,30 @@ import { Router } from "react-router-dom";
 import MatxTheme from "./MatxLayout/MatxTheme/MatxTheme";
 import AppContext from "./appContext";
 import history from "history.js";
-
 import routes from "./RootRoutes";
 import { Store } from "./redux/Store";
 import Auth from "./auth/Auth";
 import MatxLayout from "./MatxLayout/MatxLayoutSFC";
 import AuthGuard from "./auth/AuthGuard";
+import { ApolloClient, ApolloProvider, InMemoryCache, HttpLink } from '@apollo/client';
+ 
+
+
+const createApolloClient = () => {
+  return new ApolloClient({
+    link: new HttpLink({
+      uri: 'https://coa-moped.herokuapp.com/v1/graphql',
+    }),
+    cache: new InMemoryCache(),
+  });
+ };
+
+const client = createApolloClient();
+
```

Comment:
This is very good, I like how you created a client and how you provided parameters for a default uri and cache policy.  Right now there isn't a lot of clarity, but we will definitely need to stay away from hard-coded URIs soon, also there will be situations where we will need to have different cache policies for different pages.

It's not really necessary, but if you have the time, take a look at how we have done this in the past:
https://github.com/cityofaustin/atd-vz-data/blob/9c54201924df3d893a95a184efb390648eae0ed5/atd-vze/src/App.js#L31